Pipfile

```diff
@@ -49,6 +49,7 @@ kombu = ">=4.6.10,<5.0"
 importlib-metadata = "*"
 presto-python-client = ">=0.7.0"
 statsmodels = ">=0.12"
+matplotlib = ">=3.3"
```

Comment:
Do you think this should go in the `dev-packages` section since it's being used only in the script?

Reply:
Fixed!